src/main/java/me/partlysanestudios/partlysaneskies/CustomMainMenu.java

```diff
@@ -551,7 +551,7 @@ public String getDescription() {
         }
 
         public UIWrappedText createTitle(float scaleFactor, int postNum, UIComponent parent) {
-            UIComponent text = new UIWrappedText(StringUtils.colorCodes("&e" + title))
+            UIComponent text = new UIWrappedText(StringUtils.colorCodes("§e" + title))
```

Comment:
You can remove the StringUtils.colorCodes() when you already use § instead of & lol

Reply:
ok give me a sec im working on the dogfood message first

Reply:
ok done